src/YTableActionHandler.cc

```diff
@@ -0,0 +1,108 @@
+/*
+  Copyright (C) 2020 SUSE LLC
+
+  This library is free software; you can redistribute it and/or modify
+  it under the terms of the GNU Lesser General Public License as
+  published by the Free Software Foundation; either version 2.1 of the
+  License, or (at your option) version 3.0 of the License. This library
+  is distributed in the hope that it will be useful, but WITHOUT ANY
+  WARRANTY; without even the implied warranty of MERCHANTABILITY or
+  FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public
+  License for more details. You should have received a copy of the GNU
+  Lesser General Public License along with this library; if not, write
+  to the Free Software Foundation, Inc., 51 Franklin Street, Fifth
+  Floor, Boston, MA 02110-1301 USA
+*/
+
+#include "YTable.h"
+#include "YTableItem.h"
+
+#include <boost/algorithm/string.hpp>
+#include <vector>
+
+#include "YTableActionHandler.h"
+
+#define YUILogComponent "rest-api"
+#include "YUILog.h"
+
+std::function<void (YTable*)> YTableActionHandler::get_handler( YTable * widget,
+                                                                const std::string &value,
+                                                                const int &column_id,
+                                                                const int &row_id )
+{
+    // Handle case when want select row by row number
```

Comment:
when _we_ want _to_ select  _a_ row by _the_ row number  ?

Reply:
Improved that comment, thanks!